templates/fortinet_fortios_get_system_interface.textfsm

```diff
@@ -0,0 +1,37 @@
+#
+# Patrick Marc Preuss 
+# 2020-01-13 - Initial Version
+#
+# FG Version: 6.0.7
+# HW        : FG1500D
+# HA        : AP
+# VDOMS     : ENABLED
+Value Required NAME (\S+)
+Value MODE (\S+)
+Value MANAGEMENT_IP (\d+?\.\d+?\.\d+?\.\d+?)
+Value MANAGEMENT_NETMASK (\d+?\.\d+?\.\d+?\.\d+?)
+Value IP (\d+?\.\d+?\.\d+?\.\d+?)
+Value NETMASK (\d+?\.\d+?\.\d+?\.\d+?)
+Value STATUS (\S+)
+Value NETBIOS_FORWARD (\S+)
+Value TYPE (\S+)
+Value NETFLOW_SAMPLER (\S+)
+Value SFLOW_SAMPLER (\S+)
+Value SCAN_BOTNET_CONNECTIONS (\S+)
+Value SRC_CHECK (\S+)
+Value EXPLICIT_WEB_PROXY (\S+)
+Value EXPLICIT_FTP_PROXY (\S+)
+Value PROXY_CAPTIVE_PORTAL (\S+)
+Value MTU_OVERRIDE (\S+)
+Value WCCP (\S+)
+Value DROP_OVERLAPPED_FRAGMENT (\S+)
+Value DROP_FRAGMENT (\S+)
+
+Start
+  ^name:\s+${NAME}\s+mode:\s+${MODE}\s+management-ip:\s+${MANAGEMENT_IP}\s+${MANAGEMENT_NETMASK}\s+ip:\s+${IP}\s+${NETMASK}\s+status:\s+${STATUS}\s+netbios-forward:\s+${NETBIOS_FORWARD}\s+type:\s+${TYPE}\s+netflow-sampler:\s+${NETFLOW_SAMPLER}\s+sflow-sampler:\s+${SFLOW_SAMPLER}\s+scan-botnet-connections:\s+${SCAN_BOTNET_CONNECTIONS}\s+src-check:\s+${SRC_CHECK}\s+explicit-web-proxy:\s+${EXPLICIT_WEB_PROXY}\s+explicit-ftp-proxy:\s+${EXPLICIT_FTP_PROXY}\s+proxy-captive-portal:\s+${PROXY_CAPTIVE_PORTAL}\s+mtu-override:\s+${MTU_OVERRIDE}\s+wccp:\s+${WCCP}\s+drop-overlapped-fragment:\s+${DROP_OVERLAPPED_FRAGMENT}\s+drop-fragment:\s+${DROP_FRAGMENT}\s+$$ -> Record
+  ^==\s+[\s+\S+\s]$$
+  ^\s*$$
+  ^.*$$
```

Comment:
Can you remove the following from both templates?
```
  ^.*$$

EOF
```

The first one will match anything which we try to avoid and match things more explicitly. Say the data changes and instead of it erroring out on the new data, it will silently match it with that.

And I believe EOF should not be need as it's implicit. Other than those, they look ready to merge after those are removed.